client/src/components/ui/TrackingProgress.svelte

```diff
@@ -0,0 +1,91 @@
+<script lang="ts">
+    import type { PaperTrail } from '~model/api';
+    import { Status } from '../../../../model/src/snapshot';
+
+    export let trail: PaperTrail[];
+
+    $: registerDate = trail
+        .find(trail => trail.status === Status.Register)
+        ?.creation
+        .toDateString();
+    $: inSystemDate = trail
+        .find(trail => trail.status === Status.Send || trail.status === Status.Terminate)
+        ?.creation
+        .toDateString();
+    $: terminateDate = trail
+        .find(trail => trail.status === Status.Terminate)
+        ?.creation
+        .toDateString();
+</script>
+
+<section class="justify-content-between">
+    <div class="completed">
+        <p class="is-complete"></p>
+        <p>Registered<br><span>{registerDate ?? 'Pending'}</span></p>
+    </div>
+    <div class:completed={inSystemDate ?? terminateDate}>
+        <p class="is-complete"></p>
+        <p>In System<br><span>{inSystemDate ?? 'Pending'}</span></p>
+    </div>
+    <div class:completed={terminateDate}>
+        <p class="is-complete"></p>
+        <p>Terminated<br><span>{terminateDate ?? 'Pending'}</span></p>
+    </div>
+</section>
+
+<style>
+    section {
+        display: flex;
+    }
+
+    div {
+        text-align: center;
+        width: 33vw;
```

Comment:
This looks suspicious to me. Couldn't we have used CSS Grid here instead so that we can allocate a more exact `1fr 1fr 1fr`?